src/main/resources/static/css/style.css

```diff
@@ -134,3 +134,7 @@ li {
     font-size: 18px;
     padding: 10px;
 }
+
+.text-blue {
```

Comment:
careful: I think dianas branch is merge-ready and she has different css files now - you probably need to change where this goes